From 93e8084b8b0008c98abceaafcaf6c966fc811625 Mon Sep 17 00:00:00 2001
From: Zahari Doychev <zahari.doychev@linux.com>
Date: Fri, 22 May 2015 10:55:47 +0200
Subject: [PATCH] gst-plugins-good: v4l2sink use output overlay

v4l2sink plugins should use V4L2_BUF_TYPE_VIDEO_OUTPUT_OVERLAY instead of
V4L2_BUF_TYPE_VIDEO_OVERLAY to make the imx6 hardware overlay driver to work
properly otherwise the overlay window coordinates cannot be set.

Signed-off-by: Zahari Doychev <zahari.doychev@linux.com>
---
 sys/v4l2/gstv4l2sink.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sys/v4l2/gstv4l2sink.c b/sys/v4l2/gstv4l2sink.c
index 5ea374b..bf868d9 100644
--- a/sys/v4l2/gstv4l2sink.c
+++ b/sys/v4l2/gstv4l2sink.c
@@ -250,7 +250,7 @@ gst_v4l2sink_sync_overlay_fields (GstV4l2Sink * v4l2sink)
     struct v4l2_format format;
 
     memset (&format, 0x00, sizeof (struct v4l2_format));
-    format.type = V4L2_BUF_TYPE_VIDEO_OVERLAY;
+    format.type = V4L2_BUF_TYPE_VIDEO_OUTPUT_OVERLAY;
 
     if (v4l2_ioctl (fd, VIDIOC_G_FMT, &format) < 0) {
       GST_WARNING_OBJECT (v4l2sink, "VIDIOC_G_FMT failed");
-- 
2.4.1

